.ansible-lint

```diff
@@ -4,7 +4,6 @@ exclude_paths:
   - .github/
   - requirements.yml
   - src/**/cookiecutter/{{*
-  # - src/molecule_gce/cookiecutter/{{cookiecutter.molecule_directory}}
 
 skip_list:
   # Temporary skips made during migration
```

Comment:
It would be better if the skip_list can be deleted in this PR.

Reply:
Done